modules/geoipbackend/geoipbackend.cc

```diff
@@ -649,13 +649,13 @@ string GeoIPBackend::format2str(string sformat, const Netmask& addr, GeoIPNetmas
   string::size_type cur, last;
   boost::optional<int> alt, prec;
   double lat, lon;
-  time_t t = time((time_t*)NULL);
+  time_t t = time(static_cast<time_t*>(nullptr));
```

Comment:
```suggestion
  time_t t = time(nullptr);
```

Reply:
Thanks: Overlooked this one, habit of C I presume.